modules/registry/registry-core/src/main/resources/appcatalog-mysql.sql

```diff
@@ -442,6 +442,7 @@ CREATE TABLE LOCAL_SUBMISSION
 (
         RESOURCE_JOB_MANAGER_ID VARCHAR (255) NOT NULL,
         JOB_SUBMISSION_INTERFACE_ID VARCHAR (255) NOT NULL,
+        SECURITY_PROTOCOL VARCHAR (255) NOT NULL,
```

Comment:
How this is going to affect the production deployment ? Do we have a data migration mechanism in airavata ?